plugin/gcp/computeDir.go

```diff
@@ -0,0 +1,68 @@
+package gcp
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/puppetlabs/wash/plugin"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/option"
+)
+
+type computeProjectService struct {
+	*compute.Service
+	projectID string
+}
+
+type computeDir struct {
+	plugin.EntryBase
+	computeProjectService
+}
+
+const computeScope = compute.CloudPlatformScope
+
+func newComputeDir(client *http.Client, projID string) (*computeDir, error) {
+	svc, err := compute.NewService(context.Background(), option.WithHTTPClient(client))
+	if err != nil {
+		return nil, err
+	}
+	return &computeDir{
+		EntryBase:             plugin.NewEntry("compute"),
+		computeProjectService: computeProjectService{Service: svc, projectID: projID},
+	}, nil
+}
+
+// List all services as dirs.
+func (c *computeDir) List(ctx context.Context) ([]plugin.Entry, error) {
+	var entries []plugin.Entry
+	instReq := c.Instances.AggregatedList(c.projectID)
+	if err := instReq.Pages(ctx, func(instancePage *compute.InstanceAggregatedList) error {
```

Comment:
Nit: Could `err :=` move out of the if statement? Bit hard to read as it is right now.